vendor/vendor-ios/src/main/kotlin/com/malinskiy/marathon/ios/cmd/remote/SshjCommandExecutor.kt

```diff
@@ -86,11 +89,13 @@ class SshjCommandExecutor(connectionId: String,
 
         try {
             ssh = SSHClient(config)
+            if (disableHostKeyVerifier) {
```

Comment:
:+1:
Can we emit an info log message for this?

Reply:
will sure do